fe1-web/src/features/home/interface/Feature.ts

```diff
@@ -5,8 +5,9 @@ export namespace HomeFeature {
     toState: () => LaoState;
   }
 
-  export interface MainNavigationScreen {
-    name: string;
+  export interface Screen {
+    id: string;
```

Comment:
I'm not sure that `id` conveys that this is user-visible information, but it seems to be used that way 🤔

Reply:
No, `id` is (usually) not user visible, `title` is used for that :)

Before the codebase used

```typescript
<Stack.Screen
      name={SOME_NAME}
      component={}
    />
```

everywhere and I started changing it to

```typescript
<Stack.Screen
      name={SOME_NAME}
      component={}
      options={{ title: title || id }}
    />
```

So that the id is only used as a fallback display title. Apparently react-navigation does that too, i.e. if no `options.title` is specified, it shows `name`.

It is better to separate the two more since this would allow the display name to change without all .navigate() to change.